examples/online_serving/whisper_api_client.py

```diff
@@ -0,0 +1,88 @@
+# SPDX-License-Identifier: Apache-2.0
+import argparse
+import asyncio
+import json
+from subprocess import CalledProcessError, run
+
+import aiohttp
+import numpy as np
+
+AIOHTTP_TIMEOUT = aiohttp.ClientTimeout(total=6 * 60 * 60)
+SAMPLE_RATE = 16000
+
+
+def load_audio_from_file(file: str, sample_rate: int = SAMPLE_RATE):
```

Comment:
What kind of input files are we expected to support?

Reply:
I think any flac, mp3, mp4, mpeg, mpga, m4a, ogg, wav, or webm